src/timeMachine/utils/rawFluxDataTable.ts

```diff
@@ -14,6 +15,107 @@ export interface ParseFilesResult {
   maxColumnCount: number
 }
 
+export const parseFilesWithFromFlux = (
+  responses: string[]
+): ParseFilesResult => {
+  const chunks = parseChunks(responses.join('\n\n'))
```

Comment:
this throws off a million red flags

Reply:
It’s the basis for the current parsers. I can take a look at removing it since I now feel distant enough from it to see that we can probably just iterate through the results and achieve the same outcome